test/regression.uts

```diff
@@ -9646,6 +9646,81 @@ assert not a.answers(b)
 assert not (Dot11()/Dot11Ack()).answers(Dot11())
 assert (Dot11()/LLC(dsap=2, ctrl=4)).answers(Dot11()/LLC(dsap=1, ctrl=5))
 
+= RSNCipherSuite
+assert str(RSNCipherSuite()) == '\x00\x0f\xac\x04'
+rsn =  RSNCipherSuite('\x00\x0f\xac\x04')
```

Comment:
The Python3 tests are failing. You now need to pass bytes to packets by using the `b` flag, such as:
`sn =  RSNCipherSuite(b'\x00\x0f\xac\x04')`